web/cobrands/sass/_layout.scss

```diff
@@ -462,6 +462,7 @@ a.platform-logo {
 body.authpage {
   .content {
     width: 27em;
+    max-width: 27em;
```

Comment:
Can we drop the width line then; don't think it needs both?

Reply:
@dracos I just did =), let me know if you are happy for me to merge